app/preprints/index/template.hbs

```diff
@@ -0,0 +1,167 @@
+>
+    {{!HEADER}}
+    <div local-class='preprint-header'>
+        <BrandedHeader
+            @translationParent='preprints'
+            @onSearch={{action this.onSearch}}
+            @showHelp=true
+            @searchPlaceholder={{this.theme.provider.searchPlaceholder}}
+            as |branded-header|
+        >
+            {{#branded-header.lead}}
+                <div local-class='description' data-test-provider-description>
+                    {{html-safe this.theme.provider.description}}
+                </div>
+                <div>
+                    <OsfLink
+                        data-test-poweredby-button
+                        data-analytics-name='powered-by button'
+                        @href={{concat @host 'preprints'}}
+                    >
+                        {{t 'preprints.header.powered_by'}}
+                    </OsfLink>
+                </div>
+            {{/branded-header.lead}}
+            {{#branded-header.row}}
+                {{#if this.theme.provider.allowSubmissions}}
+                    <div local-class='submit-container'>
+                        <div local-class='or-container'>{{t 'preprints.header.or'}}</div>
+                        <OsfLink
+                            data-test-submit-button
+                            data-analytics-name='submit button'
+                            local-class='btn btn-success btn-lg'
+                            @href={{concat this.theme.pathPrefix 'submit'}}
+                        >
+                            {{t 'preprints.header.submit_label' documentType=this.theme.provider.content.documentType}}
+                        </OsfLink>
+                        <div local-class='example-container'>
+                            <OsfLink
+                                data-test-see-example
+                                data-analytics-name='See Example'
+                                @href={{concat '/content/' (if this.theme.provider.example this.theme.provider.example 'khbvy')}}
+                            >
+                                {{t 'preprints.header.example'}}
+                            </OsfLink>
+                        </div>
+                    </div>
+                {{/if}}
+            {{/branded-header.row}}
+        </BrandedHeader>
+    </div>
+    {{!END HEADER}}
+
+    {{!SUBJECTS}}
+    <div local-class='preprint-subjects-container'>
+        <div local-class='preprint-subjects-inner-container'>
+            <h2>{{#if this.theme.provider.additionalProviders}}
+                    {{t 'preprints.subjects.heading.provider'}}
+                {{else}}
+                    {{if this.theme.provider.hasHighlightedSubjects (t 'preprints.subjects.heading.hasHighlightedSubjects') (t 'preprints.subjects.heading.noHighlightedSubjects')}}
+                {{/if}}
+            </h2>
+            {{#if this.theme.provider.hasHighlightedSubjects}}
+                <OsfLink
+                    data-test-see-example
```

Comment:
It looks like you used `data-test-see-example` above. Can you use a unique data test selector here?